protected/tests/unit/CurationLogTest.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+class CurationLogTest extends CDbTestCase
+{
+    public function testNoLastModifiedDate()
+    {
+        $expectation = null;
+//        $this->assertEquals($expectation, CurationLog::createCurationLogEntry(8)->last_modified_date, "Last modified date created");
+        $out = CurationLog::createCurationLogEntry(8);
+        file_put_contents('test.log', var_export($out, true));
```

Comment:
Hi @rija The `test.log` was an empty file, which means there is no output from `Curation::createCurationLogEntry(8)` function, I was expecting something like:
```
creation_date = 2020-10-19;
last_modified_date = null;
dataset_id = 8;
created_by = "System";
action = "Status changed to stuff";
```
So what is the proper way to call static function for unit test?

Reply:
Hi @kencho51 

have a look here:

* https://github.com/rija/gigadb-website/blob/fuw-cicd/protected/models/CurationLog.php
* https://github.com/rija/gigadb-website/blob/fuw-cicd/protected/tests/unit/CurationLogTest.php